sources/csv/data.go

```diff
@@ -87,16 +87,16 @@ func VerifyManifest(conv *internal.Conv, tables []utils.ManifestTable) error {
 		return fmt.Errorf("no tables found")
 	}
 	missing := []string{}
```

Comment:
Please test the migrations from csv files as well as a part of the testing for name to id changes

Reply:
CSV CLI migration tested and working fine